packages/manager/apps/dedicated/client/app/account/user/security/user-security.html

```diff
@@ -541,9 +541,8 @@ <h5 data-translate="user_ipRestrictions_default_rule"></h5>
                                             for="inputWarning">
                                         <span data-translate="user_ipRestrictions_alert"></span>
                                         <span class="oui-icon oui-icon-help_circle"
-                                                data-uib-tooltip="{{ 'user_ipRestrictions_alert_information' | translate }}"
+                                                data-oui-tooltip="{{ 'user_ipRestrictions_alert_information' | translate }}"
                                                 data-tooltip-placement="bottom"
-                                                data-tooltip-append-to-body="true"
```

Comment:
This could have some impact on the display. Are we sure removing this doesn't cause side effect ?

Reply:
Does not have any visual changes. Looks the same. It creates `html` for `uib-tooltip` inside the parent of the element its called in. Since `oui-tooltip` does the same by default, there is no visual difference.

Reply:
I don’t understand your comment. You said that uib-tooltip and oui-tooltip, have the same behavior, but it’s that case, no :/

`data-tooltip-append-to-body` put the HTML for tooltip as direct child of body tag. It’s not the default behavior of a oui-tooltip, which, as you said, is going to instantiate the tooltip inside the parent.

It can create some difference and visual breaking changes, if z-index or overflow are not managed correctly for this case (especially in table for example).

If you tried it and it’s ok, let’s go ;)

Reply:
Yes, it looks good in all places i checked.

Reply:
Yes, it looks good in all places i checked.